### PR TITLE
Adding rules of procedure

### PR DESCRIPTION
**Pull request by ssteinbuss**
adding the current and valid version of the Rules of procedure for certification

closing #87

### CODE REVIEW

**Review by mohuber — APPROVED** [review]
I have fixed the formatting of the Markdown files and included the appendices, the recent changes as well as a change history. From my perspective, we could now merge this PR

**Comment by mohuber** [comment]
maybe @Sonia-IDSA or @nadjamenz could take a look and check whether I have found all issues and then provide the second review?

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:81]:
```suggestion
1. Independent and technically suitable evaluation facilities can be commissioned by the applicant to evaluate the operational environment of a participating organization or the components of a manufacturer in order to receive an IDS certificate from the IDSA certification body.
```

Comment (nadjamenz) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:81]: The old wording sounded like the EF issues the certificate.

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:119]:
```suggestion
### §2.5 Approval      
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:121]:
```suggestion
2. The International Data Spaces Association (IDSA) acts as the certification body and therefore carries out certifications in accordance with these rules of procedure.
```

Comment (nadjamenz) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:121]: The IDSA can not commission itself to act as the CB (see also next change suggestion).

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:122]:
```suggestion
3. The certification body may use its own or external experts to carry out certifications in accordance with these rules of procedure. External experts may only carry out certifications if a valid commission has been received.
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:119]: Accreditation is only the correct term for the procedure described in the annex.

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:123]:
```suggestion
4. Evaluation facilities require approval by the IDS certification body for performing evaluations in accordance with these procedural rules.
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:182]:
```suggestion
5. In the case of component certification, appropriate functional and security tests, including penetration testing, are conducted.
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:251]:
```suggestion
2. Recognized evaluation facilities are listed in a list maintained by the certification body within the meaning of § 2.6.
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:201]:
```suggestion
The evaluation report shall include at least the following information:
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:202]:
```suggestion
* a. the subject matter of the evaluation;
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:203]:
```suggestion
* b. a description of the timing and scope of all performed, on-site audits, including the locations and premises where an audit was conducted;
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:211]:
```suggestion
* j. a list of the reviewed documentation;
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (nadjamenz) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:195]: The term audit / audit report is sometimes used instead of evaluation / evaluation report in this chapter. However, at the moment 3.5 is only about the evaluation report, which should **include** information about all performed (on-site) audits. What the Rules of Procedure do not require at the moment is a separate report for each on-site audit. If this is wanted, it should be explicitly listed as its own requirement in 3.5. Otherwise, with my suggested changes, I would expect all relevant information from the on-site audit to be part of the evaluation report.

**Review by nadjamenz — COMMENTED** [review]
Final comments and change suggestions before merge approval.

**Review by Sonia-IDSA — APPROVED** [review]
Changes ok. Typo corrected.

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:81]: I agree and integrate the clarification

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:119]: accepted and adjusted

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:121]: Since this would be a significant change to the (legal) responsibilities of IDSA and (external) Certification Body, I don't want to change it at this point - we would definitely have to discuss it in the WG if a change becomes necessary.
However, as long as we understand "commissioning" more in terms of "the IDSA empowers/authorizes the CB" (in German: Bevollmächtigung) and **not** as "the IDSA pays someone to do something", we can keep the current wording from my perspective

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:122]: see comment above

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:182]: For the current revision, I would keep it as vague here as it is currently defined since we should discuss with the WG whether we want to add further details including mentioning of Source Code Audits and differentiating the Assurance Levels...

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [inline documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:391]: Adjustment necessary to correctly translate the original German sentence:
```suggestion
6.  The certification body shall take the necessary measures to ensure compliance with the IDS on the basis of its findings, if necessary on the basis of the intermediate evaluation report.
```

File: documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md
Comment (mohuber) [reply documentation/4_Perspectives_of_the_Reference_Architecture_Model/4_2_Certification_Perspective/RulesOfProcedure/rules-of-procedure.md:195]: well spotted. I would agree that so far the report from the on-site audit was planned to the a part of the overall evaluation report and have accepted the changes accordingly

**Comment by mohuber** [comment]
Hi @Sonia-IDSA 
thanks for checking the documents. Have you already taken a look at Nadjas comments during your review? I have integrated those feedback points from her review that merely affect the wording and fixed issues with the translation of the rules of procedure and additionally added capitalization for defined IDS phrases. 
If those adjustments are fine fo you, I would probably merge the pull request soon.

**Pull request merged**
